src/software/test_util/test_util.h

```diff
@@ -17,6 +18,25 @@
 #include "software/world/team.h"
 #include "software/world/world.h"
 
+#define TEST_UTIL_CREATE_MOTION_CONTROL_NO_DEST                                          \
+    [](const Robot &, const Point &) {                                                   \
+        TbotsProto::MotionControl motion_control;                                        \
+        *(motion_control.mutable_requested_destination()) = *createPointProto(Point());  \
```

Comment:
won't this cause the origin to be the requested destination each time?

Reply:
this is just a test util, not meant to be used in real gameplay